ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandQueue.java

```diff
@@ -38,6 +38,8 @@ QueuedCommandStatus enqueueCommand(
   void ensureConsumedPast(long seqNum, Duration timeout)
       throws InterruptedException, TimeoutException;
 
+  boolean empty();
```

Comment:
super nitty, but can we call it `isEmpty` so that it's not confused with "empty (drain) this queue"?